gamemodes/terrortown/gamemode/server/sv_roleselection.lua

```diff
@@ -497,7 +497,7 @@ local function SetSubRoles(plys, availableRoles, selectableRoles)
 		local rolePick = math.random(availableRolesAmount)
 		local subrole = availableRoles[rolePick]
 		local roleData = roles.GetByIndex(subrole)
-		local roleCount = tmpSelectableRoles[subrole]
+		local roleCount = selectableRoles[subrole]
```

Comment:
so you editing the roles list directly? That's the opposite we talked about. ^^

Reply:
Yeah I know, but it works so smoothly with the local list..... It's direct and can be easily edited.

Reply:
And what your marked there is just a performance increase. If needed this could stay as a copied table. Only the force needs to change it, the rest works lile this too.